packages/@atjson/source-html/src/converter/social-embeds.ts

```diff
@@ -99,6 +118,18 @@ export default function (doc: Document) {
         }
       }
 
+      if (
+        canonicalURL &&
+        blockquote?.attributes?.dataset &&
+        blockquote?.attributes?.class == "instagram-media"
+      ) {
+        const instaAttributes = checkInstagramCaption(
+          canonicalURL?.attributes,
+          blockquote.attributes.dataset
+        );
+        canonicalURL.attributes = instaAttributes;
+      }
```

Comment:
Please write a test to illustrate the change you are making. That test should show the new html embed code and verify that the converter produces an annotation with the correct attributes

Reply:
Also, you will need to update the InstagramEmbed annotation definition since currently `captioned` is not one of its defined attributes

Reply:
ex:
```
// packages/@atjson/offset-annotations/src/annotations/iframe-embed.ts
export class IframeEmbed<ExtendedAttributes = {}> extends BlockAnnotation<
  {
    url: string;
    ...
  } & ExtendedAttributes
> {
  ...
}


// packages/@atjson/offset-annotations/src/annotations/instagram-embed.ts
export class InstagramEmbed extends IframeEmbed<{ captioned?: boolean }> {
  ...
}

```